src/re/main.c

```diff
@@ -1047,7 +1047,7 @@ main(int argc, char *argv[])
 	}
 
 	if (generate_bounds > 0) {
-		if (!fsm_generate_matches(fsm, generate_bounds, fsm_generate_cb_printf_escaped, &opt)) {
+		if (!fsm_generate_matches(fsm, generate_bounds, 0, fsm_generate_cb_printf_escaped, &opt)) {
```

Comment:
would make sense to expose this to the cli of course (and also for fsm(1)), but that can wait